### PR TITLE
Remove separate run of metrics ostream example

### LINKED ISSUE #1952: Run metrics_ostream_example separately in bazel.asan/tsan?
In below CI script, we run metrics_ostream_example separately after bazel run. Is this necessary or can be removed?

https://github.com/open-telemetry/opentelemetry-cpp/blob/d56a5c702fc2154ef82400df4801cd511ffb63ea/ci/do_ci.sh#L296

### PR DESCRIPTION
**Pull request by ThomsonTan**
Fixes #1952 

## Changes

It is no longer needed to run the metrics ostream example in CI.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

**Review by lalitb — APPROVED** [review]
Thanks.

File: ci/do_ci.sh [hunk 1]
```diff
@@ -321,14 +321,12 @@ elif [[ "$1" == "bazel.nortti" ]]; then
 elif [[ "$1" == "bazel.asan" ]]; then
   bazel $BAZEL_STARTUP_OPTIONS test --config=asan $BAZEL_TEST_OPTIONS_ASYNC //...
   bazel $BAZEL_STARTUP_OPTIONS run --config=asan $BAZEL_TEST_OPTIONS_ASYNC \
-  //examples/metrics_simple:metrics_ostream_example > /dev/null
```
Comment (marcalff) [inline ci/do_ci.sh:324]: Only half the command is removed.

Delete this line as well ?

```
bazel $BAZEL_STARTUP_OPTIONS run --config=asan $BAZEL_TEST_OPTIONS_ASYNC \
```

Comment (ThomsonTan) [reply ci/do_ci.sh:324]: Yes, thanks for the catch.

Comment (lalitb) [reply ci/do_ci.sh:324]: Good catch.

**Review by marcalff — APPROVED** [review]

**Pull request merged**